### DISCUSSION TITLE
Web3Auth provides different PubKey for the same email

### DISCUSSION THREAD

**Discussion by jackbl1** [Web3Auth/Web3Auth#1202] [Help]
Hi,

I have been using my email jackbloomfeld1@gmail.com to sign into web3auth for quite a few months. However now it seems that when I sign into web3auth with the same email it provides me a different public key. I've lost all of the content that was on my previous wallet. Is this a known issue? Any quick fix?

Thanks

**Comment by Saviour1001** [comment]
You can visit [app.openlogin.com](https://app.openlogin.com/) to check the authorised apps and some other details.

**Reply by jackbl1** [reply]
How would this help?

When I logged in with web3auth last week, my wallet had public key X and a few NFTs. When I log in to web3auth today, my wallet has public key Z and no longer has the NFTs. Is there a way for you to fix this problem?
